### DISCUSSION TITLE
What does the --name option in `gitlab project-merge-request-approval-rule list`?

### DISCUSSION THREAD

**Discussion by MaximeCheramy** [python-gitlab/python-gitlab#2132] [Q&A]
The GitLab API doesn't seem to have a name option: https://docs.gitlab.com/ee/api/merge_request_approvals.html#get-merge-request-level-rules

**Comment by nejch** [comment]
This is a bug @MaximeCheramy, thanks for the report. It'll be fixed in https://github.com/python-gitlab/python-gitlab/issues/2135 with https://github.com/python-gitlab/python-gitlab/pull/2136.
